### PR TITLE
Fix some UI design and text for look more professional

### PR DESCRIPTION
**Pull request by iamtariqul**
Change UI design

Screenshot from 2021-03-21 21-30-18

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -12225,9 +12225,9 @@
       }
     },
     "react-native-elements": {
-      "version": "3.3.0",
-      "resolved": "https://registry.npmjs.org/react-native-elements/-/react-native-elements-3.3.0.tgz",
-      "integrity": "sha512-GNvS7/26WNo0ovZ2vQWRFWJxhyDeBrGakFSX0aCey69B6KtZQh8gaFGF6CutlRjMzVfZTHGU2+0NS8A0dNWhAA==",
+      "version": "3.3.2",
+      "resolved": "https://registry.npmjs.org/react-native-elements/-/react-native-elements-3.3.2.tgz",
+      "integrity": "sha512-bDWQt0WEuJEOrwzSYx0mVBV3aoRtJAsK0Kc2sioMsshSEhPOZK17Ccx+EvLBSpZ9uWxPyx42Z7hUH/8dQNu67g==",
       "requires": {
```
Comment (Uyadav207) [inline package-lock.json:12231]: These changes are conflicting with the upstream

**Review by Uyadav207 — CHANGES_REQUESTED** [review]
Can you resolve the conflict? @tstariqul.

-  Also kindly confirm with the maintainer, if these changes are required...?
- To resolve conflict kindly syncing the fork with upstream.

**Comment by piyushdevOP** [comment]
[Modified] UI Text Style Changed for Professional Look #93
@flyingcircle  @pranshuchittora Please look into it .
Should I add effects to the blocks to make it more interactive ?

**Comment by Uyadav207** [comment]
@tstariqul, Please resolve the conflict in these files... and see if is the PR isn't duplicate to #93  
```
package-lock.json
src/containers/Drawer/index.jsx
src/pages/home/index.jsx
```

**Comment by pranshuchittora** [comment]
Oops!
Pls pull the new changes & resolve the conflicts.

**Pull request closed without merging**